unattended_installer/cert_tool/certFunctions.sh

```diff
@@ -23,13 +27,40 @@ function cert_checkOpenSSL() {
     fi
 }
 
+function cert_checkRootCA() {
+    if  [[ -n ${rootca} ]]; then
+        read -p 'root-ca.key: ' rootcakey
+        while  [[ -z ${rootcakey} ]]; do
+            common_logger "You have to also enter the root-ca.key"
+            read -p 'root-ca.key: ' rootcakey
+            ((c++)) && ((c==3)) && common_logger -e "You have not entered a root-ca.key, exiting" && cert_cleanFiles && exit 1 
+        done
+        if [[ -e ${rootca} ]]; then
+            eval "cp ${rootca} ${base_path}/certs/root-ca.pem ${debug}"
+        else
+            common_logger -e "The file ${rootca} does not exists"
+            cert_cleanFiles
+            exit 1
+        fi
+        if [[ -e ${rootcakey} ]]; then
+            eval "cp ${rootcakey} ${base_path}/certs/root-ca.key ${debug}"
+        else
+            common_logger -e "The file ${rootcakey} does not exists"
+            cert_cleanFiles
+            exit 1
+        fi
+    else
+        cert_generateRootCAcertificate
+    fi
+}
+
```

Comment:
It doesn't convince me to ask the user for the file in this way, wouldn't it be better to do it by parameter?